src/components/chatbox/ai-feature-toast.tsx

```diff
@@ -0,0 +1,61 @@
+import { useEffect } from 'react'
+import Cookies from 'js-cookie'
+import { IconWand24 } from '@hashicorp/flight-icons/svg-react/wand-24'
+
+import { toast, ToastColor } from 'components/toast'
+import useAuthentication from 'hooks/use-authentication'
+
+const COOKIE_IGNORE_AI_TOAST = 'dev-dot-ignore-ai-toast'
+const COOKIE_HAS_SEEN_AI_TOAST = 'dev-dot-has-seen-ai-toast'
+const COOKIE_EXPIRES_AT = new Date('12/25/2024') // arbitrary date
+
+const TOAST_AUTO_DISMISS_MS = 15000
+
+export function AIFeatureToast() {
+	const { session } = useAuthentication()
+
+	useEffect(() => {
+		// skip toast if we're past the expiration date
+		if (new Date() > COOKIE_EXPIRES_AT) {
+			return
+		}
+
+		// skip toast if user has previously dismissed it
+		if (Cookies.get(COOKIE_IGNORE_AI_TOAST)) {
+			return
+		}
+
+		// skip toast if it's already been seen
+		if (Cookies.get(COOKIE_HAS_SEEN_AI_TOAST)) {
+			return
+		}
+
+		if (session?.meta.isAIEnabled) {
+			toast({
+				color: ToastColor.highlight,
+				icon: <IconWand24 />,
+				title: 'Welcome to the Developer AI closed beta',
+				description: 'Try it out in our cmd+K menu!',
```

Comment:
@dstaley (cross-platform-ignorance here) - what would you expect instead of ( <kbd>Cmd</kbd> + <kbd>K</kbd> ) on Windows, and Linux?

---
> [!IMPORTANT]
> TODO(@thiskevinwang) - work with @kellyschummer on this copy

Reply:
Either `Cmd/Ctrl-K` or `Command/Control-K`.

Reply:
Tentative result:
- Request from design in https://app.asana.com/0/0/1205523765153660/f
- Requires a [trivial] type change on `ToastDisplayProps.description` from `string` to `ReactNode`
- Reuses the same SVGs in our top search bar.

<img width="401" alt="CleanShot 2023-09-21 at 09 43 38@2x" src="https://github.com/hashicorp/dev-portal/assets/26389321/01068409-a67e-47ab-9a83-bc703d58f5de">

<img width="390" alt="CleanShot 2023-09-21 at 09 43 26@2x" src="https://github.com/hashicorp/dev-portal/assets/26389321/37f3aa1c-6956-4a25-ae80-eee45fed0bd6">

Reply:
Fixed with 1705b03a1